util/libeay.num

```diff
@@ -4370,7 +4370,7 @@ DH_compute_key_padded                   4732	EXIST::FUNCTION:DH
 ECDSA_METHOD_set_sign                   4733	EXIST::FUNCTION:ECDSA
 CMS_RecipientEncryptedKey_cert_cmp      4734	EXIST:!VMS:FUNCTION:CMS
 CMS_RecipEncryptedKey_cert_cmp          4734	EXIST:VMS:FUNCTION:CMS
-DH_KDF_X9_42                            4735	EXIST::FUNCTION:DH
+DH_KDF_X9_42                            4735	EXIST::FUNCTION:CMS,DH
```

Comment:
> Why wouldn't it be impossible to omit something mid-series if there was *legitimate* need for that?

"legitimate"?  When would it be "legitimate" to remove a whole API and ABI mid-series?  I cannot imagine a responsible general packager doing so.
Sure, someone or some organization might want to do so for their own purposes, *but then they wouldn't use that API / ABI to start with, would they*?

So what I get from what you're saying is that just because we added a *possibility*, we are responsible for someone else using it?  I disagree with that line of thinking, but if this is a team opinion, then I'll close this PR, and we'll have to say that "sorry, we don't support `no-cms`" in 1.0.2.  I'm fine with that move too (but think we should document that clearly)